BDArmory/Ammo/PooledBullet.cs

```diff
@@ -561,6 +560,8 @@ private void LeapfrogVelocityHalfStep(float period)
         /// <returns></returns>
         Vector3 GetDragAdjustedVelocity()
         {
+            atmosphereDensity = (float)FlightGlobals.getAtmDensity(FlightGlobals.getStaticPressure(currentPosition), FlightGlobals.getExternalTemperature(currentPosition)); //moving this here so it's not getting called 2x a tick in LeapFrogVelocityHalfStep
```

Comment:
You also need to call `UpdateDragEstimate` so that the new `atmDensity` is used.